Spotify Bluetooth Headset Listener.xcodeproj/project.pbxproj

```diff
@@ -398,7 +398,7 @@
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = "Spotify Bluetooth Headset Listener/Spotify Bluetooth Headset Listener-Prefix.pch";
 				INFOPLIST_FILE = "Spotify Bluetooth Headset Listener/Spotify Bluetooth Headset Listener-Info.plist";
-				MACOSX_DEPLOYMENT_TARGET = 10.6;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```

Comment:
Might have to do with your build environment.  There is a new version rolling out soon-ish:  https://travis-ci.org/jguice/mac-bt-headset-fix/builds

I'll test it out on Yosemite and see how it goes :smile: